src/app/components/BackButton/BackButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useHistory } from 'react-router';
+import styles from './BackButton.module.css';
+
+type BackProps = {
+  icon: React.SVGProps<SVGSVGElement>;
+};
+
+export default function BackButton({ icon }: BackProps): JSX.Element {
```

Comment:
Do you plan to use different `icon` here? Otherwise, you could directly import `BackButtonIcon` in this module.

Reply:
You're right. there are no plans to change the icon. I adjusted it. Thanks for your review!